cmd/internal/labeler/labeler.go

```diff
@@ -184,8 +184,16 @@ func GetMemoryAmount(sysfsDrmDir, gpuName string, numTiles uint64) uint64 {
 }
 
 // GetTileCount reads the tile count.
-func GetTileCount(sysfsDrmDir, gpuName string) (numTiles uint64) {
-	filePath := filepath.Join(sysfsDrmDir, gpuName, "gt/gt*")
+func GetTileCount(cardPath string) (numTiles uint64) {
+	filePath := ""
+
+	if _, err := os.Stat(filepath.Join(cardPath, "gt")); err == nil {
```

Comment:
`Glob()` takes a list of matches so it should be possible to `matchPaths := []string{filepath.Join(cardPath, "gt/gt*"), filepath.Join(cardPath, "tile?/gt*"}`

Reply:
Anyhow, the logic required changes as the tile counting for xe was too broad. It shouldn't count gt* files under the tile* directory.